Packs/AWS-CloudTrail/Integrations/AWS-CloudTrail/AWS-CloudTrail.py

```diff
@@ -1,30 +1,31 @@
-import demistomock as demisto
-from CommonServerPython import *
-from CommonServerUserPython import *
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
 import boto3
 from botocore.config import Config
 from botocore.parsers import ResponseParserError
 import urllib3.util
-from datetime import datetime
-from typing import Any
+from datetime import datetime, date
 
 # Disable insecure warnings
 urllib3.disable_warnings()
 
-PARAMS = demisto.params()
-AWS_DEFAULT_REGION = PARAMS['defaultRegion']
-AWS_ROLE_ARN = PARAMS['roleArn']
-AWS_ROLE_SESSION_NAME = PARAMS['roleSessionName']
-AWS_ROLE_SESSION_DURATION = PARAMS['sessionDuration']
+AWS_DEFAULT_REGION = demisto.params()['defaultRegion']
+AWS_ROLE_ARN = demisto.params()['roleArn']
+AWS_ROLE_SESSION_NAME = demisto.params()['roleSessionName']
+AWS_ROLE_SESSION_DURATION = demisto.params()['sessionDuration']
 AWS_ROLE_POLICY = None
-AWS_ACCESS_KEY_ID = PARAMS.get('credentials', {}).get('identifier') or PARAMS.get('access_key')
-AWS_SECRET_ACCESS_KEY = PARAMS.get('credentials', {}).get('password') or PARAMS.get('secret_key')
-VERIFY_CERTIFICATE = not PARAMS.get('insecure', True)
+AWS_ACCESS_KEY_ID = demisto.params().get('credentials', {}).get('identifier') or demisto.params().get('access_key')
+AWS_SECRET_ACCESS_KEY = demisto.params().get('credentials', {}).get('password') or demisto.params().get('secret_key')
+VERIFY_CERTIFICATE = not demisto.params().get('insecure', True)
```

Comment:
why using demisto.params() so many times? let's revert to the old behaviour where we used demisto.params one and then used this variable